Lib/test/test_gdb.py

```diff
@@ -849,7 +849,7 @@ def test_pycfunction(self):
         # Various optimizations multiply the code paths by which these are
         # called, so test a variety of calling conventions.
         for py_name, py_args, c_name, expected_frame_number in (
-            ('gmtime', '', 'time_gmtime', 1),  # METH_VARARGS
+            ('gmtime', '', 'time_gmtime', 2),  # METH_VARARGS
```

Comment:
This change is not correct. The whole point of this test is that we're testing a `METH_VARARGS` function. See #14330 for a fix.

Reply:
I'm sorry. I did not realize that the Argument Clinic would use `METH_FASTCALL` for `gmtime()`.